libcst/codemod/visitors/tests/test_remove_imports.py

```diff
@@ -797,3 +797,17 @@ def visit_ImportFrom(self, node: cst.ImportFrom) -> None:
             after,
             RemoveImportTransformer(CodemodContext()).transform_module(module).code,
         )
+
+    def test_remove_comma(self) -> None:
+        """
+        Trailing commas should be removed if and only if the last alias is removed.
+        """
+        before = """
+            from m import (a, b,)
```

Comment:
In the case of `from n import (c, d,)` and remove `("n", "d", None)`, the trailing comma should still exist `from n import (c,)` (to make it still a valid Tuple.
Can you also test it?

Reply:
Are you sure? `from n import (c)` is perfectly valid Python (and will be reformatted to `from n import c` by the formatter)

Reply:
You are right. Both `from a import (b)` and `from a import (b,)` are valid code and the trailing comma should be removed in the codemod.